### PR TITLE
BD-3331. Staking wallet view values

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: client/src/pages/WalletPage/styles/TransactionList.module.scss [hunk 1]
```diff
@@ -124,7 +124,7 @@
         align-items: center;
         justify-content: space-between;
         display: grid;
-        grid-template-columns: 130px 180px 250px 100px;
+        grid-template-columns: 200px 80px 60px;
```
Comment (trukhilio) [inline client/src/pages/WalletPage/styles/TransactionList.module.scss:127]: Screenshot 2022-03-07 at 19 16 39

Comment (trukhilio) [reply client/src/pages/WalletPage/styles/TransactionList.module.scss:127]: fix ui

**Review by trukhilio — COMMENTED** [review]
see comments

**Comment by andreyvEze** [comment]
@trukhilio Снимок экрана 2022-03-07 в 22 21 16

**Review by NikolasStelmakh — APPROVED** [review]

**Pull request merged**